docs/dashboard/sdk/python.md

```diff
@@ -80,25 +80,54 @@ def handler(event, context):
 all invocations associated with that tag. To provide extra context, you can specify a tag value to optionally filter on. If you're accustomed to
 logging out a debugging object, you can pass a third `custom` attribute that will be surfaced in the dashboard as well.
 
-The `tagEvent` function is available on either the `context.serverless_sdk` or on the
+The `tag_event` function is available on either the `context.serverless_sdk` or on the
 module imported from `'./serverless_sdk'`.
 
-Here is an example of how to use it from the `context.serverlessSdk` object:
+Here is an example of how to use it from the `context.serverless_sdk` object:
 
 ```python
 def hello(event, context):
     # ... set up some state/custom logic
-    context.serverless_sdk.tagEvent('customer-id', event.body.customerId, { 'demoUser': 'true', 'freeTrialExpires': '2020-09-01' })
+    context.serverless_sdk.tagEvent(
+        'customer-id',
+        event.body.customerId,
+        { 'demoUser': 'true', 'freeTrialExpires': '2020-09-01' }
+    )
     return {
         'statusCode': 500,
         'body': '{"name": "bob"}',
     }
 ```
+
+# `set_endpoint`
+
+Faced with practical considerations (a big one being CloudFormation stack resource limit), developers often reach for a single function solution with routing being handled by the application layer. This is typically accomplished by leveraging the [serverless-wsgi](https://github.com/logandk/serverless-wsgi) plugin to deploy existing WSGI applications (Flask/Django/Pyramid etc). Rolling your own custom router is another option as well.
+
+An unfortunate downside of this approach is the loss of visibility into the mapped route for invocations. Instead, you're left with either the catch-all API Gateway resource path (`/{proxy+}`) or the raw request url itself (e.g. `/org/foo/user/bar/orders`). Neither of which are conducive for exploration and debugging invocations. The former is obviously not very useful and the latter wouldn't let you group invocations by their routed endpoints to bubble up say, performance issues.
+
+To alleviate these issues, the `setEndpoint` SDK function allows you to explicitly set the application routed endpoint for an invocation. Once set, invocations can be explored and inspected by endpoint in the Dashboard. Like the other SDK methods, `setEndpoint` is available on either the context object: `context.serverless_sdk`.
+
+```python
+def handler(event, context):
+  context.serverless_sdk.set_endpoint('/api/foo')
+  # application code...
+```
+
+You can also import the function from `serverless_sdk`
+
+```python
+from serverless_sdk import set_endpoint
+def handler(event, context):
+  set_endpoint('/api/foo')
+  # application code...
+```
+
+Furthermore, if you're using deploying a [Flask](https://flask.palletsprojects.com/en/1.1.x/) application, there's no explicit instrumentation needed - the SDK will set the endpoint for you automatically!
```

Comment:
Same comment applies here for "Flask" as it does for "lambda-api" and "express-js".